spec/models/user_spec.rb

```diff
@@ -6,6 +6,37 @@
     allow(User).to receive(:chimp_subscribe).and_return(true)
   end
 
+
+  describe 'contact information' do
+
+    context 'with valid fields' do
+      it 'should accept a two letter state' do
+        user = build(:user, state: 'PA')
+        user.valid?
+        expect(user).to be_valid
```

Comment:
so what `be_` does is takes whatever you put after it and calls xxx? on it, so if your model had a method called `horsey?` you could do

```
expect(user).to be_horsey
```

and it would get called.  

Short answer is you shouldn't need that first `user.valid?`